altos-rust/port/cortex-m0/src/peripheral/usart/baudr.rs

```diff
@@ -0,0 +1,71 @@
+// Daniel Seitz and RJ Russell
+
+use super::super::Register;
+use super::defs::*;
+
+#[derive(Copy, Clone)]
+pub struct UsartBRR {
+    brr: BRR,
+}
+
+pub enum BaudRate {
+    Rate4800,
```

Comment:
Maybe instead of `Rate4800...` the variants can be called `Hz4800...`. So that way you don't have to type `BaudRate::Rate...`